tests/go.mod

```diff
@@ -14,7 +14,7 @@ require (
 	github.com/golang/groupcache v0.0.0-20200121045136-8c9f03a8e57e // indirect
 	github.com/golang/mock v1.4.0 // indirect
 	github.com/hashicorp/golang-lru v0.5.3 // indirect
-	github.com/kubevirt/cluster-network-addons-operator v0.43.0
+	github.com/kubevirt/cluster-network-addons-operator v0.42.4
```

Comment:
This is only temporary right?

Reply:
yes, #993 hasn't been merged yet.

Reply:
#993 is merged now, reminder to remove it :)